src/Common/MessageBroker/Publisher.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gaming\Common\MessageBroker;
+
+use Gaming\Common\MessageBroker\Exception\MessageBrokerException;
+use Gaming\Common\MessageBroker\Model\Message\Message;
+
+interface Publisher
+{
+    /**
+     * @throws MessageBrokerException
+     */
+    public function publish(Message $message): void;
+
+    /**
+     * @throws MessageBrokerException
```

Comment:
Describe the semantics in a comment as `flush` is a pretty generic name. While it can be used with RabbitMQ for publisher confirms, it can be used with another message broker, e.g. NATS, to ensure that packets are flushed to the network.